### ISSUE TITLE
Why not archiving this project?

### ISSUE THREAD

**Issue by joshlopes** [FriendsOfPHP/Goutte#462]
If all this does now is extending symfony browserkit client, why not just archive it? I was so confused by this library and goutte-driver and mink-extension, the whole thing actually does not work. 

I think we need a new factory in mink-extension that initiates this browserkit client version opposing to all this wrapper madness. 

I've created a PR and made the goutte-driver work but is completely misleading and parameters are actually not being sent either. So, happy to fix the issue but will need some light shed on this issue and the best way to tackle it.

**Comment by edomato** [comment]
Hi, BrowserKit is just one component of this package, it also depends (and wraps) other components so you don't have to do it yourself.

Or maybe I'm missing something here? :smile: 

Greets

**Comment by Sairony** [comment]
Yeah I don't get it, what's this library actually doing? Taking a quick look at the code base the entirety of this project seems to be:

`class Client extends HttpBrowser`

Am I missing something?

**Comment by mvhirsch** [comment]
> You can now use any of the methods shown in this article to extract information, click links, submit forms, etc. This means that you no longer need to use a dedicated web crawler or scraper such as [Goutte](https://github.com/FriendsOfPHP/Goutte):

Exactly.

https://symfony.com/doc/current/components/browser_kit.html#making-external-http-requests
